src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CapiHelper.cs

```diff
@@ -698,7 +698,7 @@ private static void ValidateCspFlags(CspProviderFlags flags)
                                             CspProviderFlags.UseUserProtectedKey);
                 if ((flags & keyFlags) != CspProviderFlags.NoFlags)
                 {
-                    throw new ArgumentException(SR.Format(SR.Argument_InvalidValue, Convert.ToString(flags)));
+                    throw new ArgumentException(SR.Argument_InvalidValue, nameof(flags));
```

Comment:
+1 for ArgumentException needs an actual value 👍 Yeah I will use the Format + argumentname so a mix of both. thanks Dan!